package.json

```diff
@@ -75,7 +75,7 @@
     ]
   },
   "dependencies": {
-    "@internetarchive/field-parsers": "^0.1.3",
+    "@internetarchive/field-parsers": "0.1.3-alpha1",
```

Comment:
Q: What was updated in the field parsers?

Reply:
adding new mediatype type `search`.